docs/src/components/Utils/ApiTable/ApiTable.component.tsx

```diff
@@ -93,7 +110,13 @@ const Table = styled('table')`
         }
 
         pre {
-            ${InlineCodeStyle};
+            display: inline;
+            background-color: rgba(27, 31, 35, 0.05);
+            font-family: 'SFMono-Regular', Consolas, Liberation Mono, Menlo,
+                Courier, monospace;
+            border-radius: 0.1875rem;
```

Comment:
We do have entries in the theme for the radius. This is a `3px` value and we don't have an entry for that but `base` resolves to `4px` which may be more congruent with the theme.